Documentation/git-sparse-checkout.txt

```diff
@@ -86,6 +86,56 @@ negate patterns. For example, to remove the file `unwanted`:
 ----------------
 
 
+## CONE PATTERN SET
+
+The full pattern set allows for arbitrary pattern matches and complicated
```

Comment:
I was thrown by the intro for the "cone" method talking about the "full" set.

Perhaps a L2 header for "Full/.gitignore algorithm" and then a L2 for the "Cone algorithm".